spacer/tests/test_train_utils.py

```diff
@@ -90,34 +117,38 @@ def test_too_few_classes(self):
                                   feature_dim,
                                   feature_loc)
 
-        self.assertRaises(ValueError, train, labels, feature_loc, num_epochs)
+        for clf_type in ['LR', 'MLP']:
+            self.assertRaises(ValueError, train, labels, feature_loc,
+                              num_epochs, clf_type)
 
 
 class TestEvaluateClassifier(unittest.TestCase):
 
     def test_simple(self):
         feature_loc = DataLocation(storage_type='memory', key='')
         train_data = make_random_data(10, [1, 2], 4, 5, feature_loc)
-        clf, _ = train(train_data, feature_loc, 1)
+        for clf_type in ['LR', 'MLP']:
```

Comment:
Better to iterate over `config.CLASSIFIER_TYPES` since 1) this test will test new stuff you add in the future. 2) you help other coders realize this list is defined in the config.

Reply:
you're right, working on this